test/test_utils.h

```diff
@@ -173,6 +174,25 @@ namespace testing
     EXPECT_EQ(3, _msg.z);
   }
 
+  /// \brief Create a message used for testing.
+  /// \param[out] _msg The message populated.
+  void createTestMsg(rosgraph_msgs::Clock &_msg)
+  {
+    _msg.clock.sec  = 1;
+    _msg.clock.nsec = 2;
+  }
+
+  /// \brief Compare a message with the populated for testing.
+  /// \param[in] _msg The message to compare.
+  void compareTestMsg(const rosgraph_msgs::Clock &_msg)
+  {
+    rosgraph_msgs::Clock expected_msg;
+    createTestMsg(expected_msg);
```

Comment:
`expected_msg` is not being used in this function?

Reply:
fixed